index.js

```diff
@@ -1,7 +1,83 @@
+
 // Iteration 1: Names and Input
+const hacker1 = "João Madeira"
+const hacker2 = "Pedro Nogueira"
+console.log("The driver name is ",hacker1,", and the navigator name is ",hacker2) 
```

Comment:
When you call the values of the variables you defined earlier, you should use `string interpolation` and it's syntax.
Something like this:
```console.log(`The driver name is ${hacker1}, and the navigator name is ${hacker2}`)```